src/code/z_player_lib.c

```diff
@@ -1406,47 +1452,45 @@ s32 Player_IsHoldingTwoHandedWeapon(Player* player) {
     return false;
 }
 
-s32 Player_ActionToBottle(Player* player, PlayerItemAction itemAction) {
-    s32 bottle = itemAction - PLAYER_IA_BOTTLE;
+PlayerBottle Player_BottleFromIA(Player* player, PlayerItemAction itemAction) {
+    PlayerBottle bottle = GET_BOTTLE_FROM_IA(itemAction);
 
-    // Relies on bottle-related item actions to be contiguous
```

Comment:
Shouldn't this comment live *somewhere*? Or do you think the current system more-or-less implies this?

Reply:
I added it to the macro define